README.md

```diff
@@ -10,57 +10,48 @@ see [2024-03-13](meetings/2024/2024-03-13.md) and [the backlog](meetings/backlog
 
 ## Meetings
 
-Meetings are bi-weekly, alternating between an APAC-friendly time and an EMEA-friendly time. The meetings appear on the [TC39 private calendar](https://github.com/tc39/Reflector#tc39-private-calendar).
+We meet weekly.
+The meetings appear on the [TC39 private calendar](https://github.com/tc39/Reflector#tc39-private-calendar).
 
-Meeting link: <https://meet.google.com/rwh-opnw-cnk>
+Meeting link: <https://agoric.zoom.us/j/7965396534?pwd=UlJ6aXZLdnVwdlZJRko4SjBtQ1dZdz09>
 
 <!-- DST below -->
 
-### 2nd Wednesday / Thursday each month (EMEA-friendly)
+### Every Wednesday / Thursday each month (EMEA-friendly)
 
 |              |                 |
 | -----------: | --------------- |
 | US / Central | 12:00 Wednesday |
 |          UTC | 17:00 Wednesday |
 |        China | 01:00 Thursday  |
 
-### 4th Tuesday / Wednesday each month (APAC-friendly)
-
-|              |                 |
-| -----------: | --------------- |
-| US / Central | 20:00 Tuesday   |
-|          UTC | 01:00 Wednesday |
-|        China | 09:00 Wednesday |
-
 <!-- not DST below -->
 
 <!--
-### 2nd Wednesday / Thursday each month (EMEA-friendly)
+### Every Wednesday / Thursday each month (EMEA-friendly)
 
 |              |                 |
 | -----------: | --------------- |
 | US / Central | 12:00 Wednesday |
 |          UTC | 18:00 Wednesday |
 |        China | 02:00 Thursday  |
-
-### 4th Tuesday / Wednesday each month (APAC-friendly)
-
-|              |                 |
-| -----------: | --------------- |
-| US / Central | 20:00 Tuesday   |
-|          UTC | 02:00 Wednesday |
-|        China | 10:00 Wednesday |
 -->
 
 ## Folks
 
-- Convenors: [Chris de Almeida](https://github.com/ctcpip), [Jordan Harband](https://github.com/ljharb)
-- Speaker: [Michael Ficarra](https://github.com/michaelficarra)
-- Secretaries: [Samina Husain](https://github.com/SaminaHusain), [Istvan Sebestyen](https://github.com/ecmageneva)
+- Convenors:
+  [Chris de Almeida](https://github.com/ctcpip),
+  [Jordan Harband](https://github.com/ljharb),
+  [Kris Kowal](https://github.com/kriskowal)
```

Comment:
technically i think this has to wait for plenary, so maybe:
```suggestion
  [Kris Kowal](https://github.com/kriskowal) (presumptive)
```
or something?

Reply:
I wrote “pending plenary”

Reply:
I'll add this in the aforementioned follow on PR